src/pages/Governance/VotingWallet/index.tsx

```diff
@@ -84,47 +89,69 @@ export const VotingWalletUi: React.FC<VotingWalletUiProps> = ({
 
   const previouslyDelegated = !!delegate;
   const userHasLockedXVS = userStakedMantissa.isGreaterThan(0);
+  const showDepositXvs = !isDataLoading && connectedWallet && !userHasLockedXVS;
+  const showDelegateButton =
+    !isDataLoading && connectedWallet && userHasLockedXVS && voteProposalFeatureEnabled;
+  const shouldApplyMarginToTotalLocked =
+    voteProposalFeatureEnabled || !connectedWallet || showDepositXvs;
+
   return (
-    <div css={styles.root}>
-      <Typography variant="h4">{t('vote.votingWallet')}</Typography>
+    <div className="flex flex-1 flex-col lg:ml-4">
+      <h4 className="m-0 text-lg font-semibold leading-normal">{t('vote.votingWallet')}</h4>
+
+      {!voteProposalFeatureEnabled && (
+        <NoticeInfo
+          className="mt-4 w-full md:mt-6"
+          data-testid={TEST_IDS.votingDisabledWarning}
+          title={t('vote.multichain.votingOnlyEnabledOnBsc')}
+          description={
+            <Button
+              className="h-auto"
+              variant="text"
+              onClick={() => switchChain({ chainId: governanceChain.id })}
+            >
+              {t('vote.multichain.switchToBsc')}
+            </Button>
+          }
+        />
+      )}
 
-      <Paper css={styles.votingWalletPaper}>
-        <div css={styles.votingWeightContainer}>
-          <Typography variant="body2" css={styles.subtitle} className="text-grey">
+      <Card className="mt-6 flex flex-col items-start justify-start p-6 sm:flex-row sm:items-center sm:justify-between lg:flex-col lg:items-start">
+        <div className="border-r-delimiter pb-4 sm:border-r sm:pb-0 sm:pr-10 lg:border-r-0 lg:pb-4">
+          <p className="m-0 text-base font-semibold leading-normal tracking-[0.3px] text-grey sm:text-sm md:text-base">
```

Comment:
No need for the  `leading-normal` and `m-0` classes. Also, I'd remove the `tracking-[0.3px]` props (the designs don't seem to include it, and I'd rather keep texts consistent).

Reply:
Just to be sure: `tracking-[0.3px]` - `letter-spacing: 0.3px` is currently set in this element in the prod app. As you said, this is not present in the designs, so I removed it, although the final result is slightly different

Reply:
Ah I see, makes sense. Let's follow the designs anyway and remove these (as you did).